app/api/controllers/video.py

```diff
@@ -217,3 +222,22 @@ def post(self, id):
         if note:
             response = {"message": note}, 201
         return response
+
+
+@video_ns.route("/stream/<string:videoId>/<int:format>")
+class GetVideoStream(Resource):
+    @token_required
+    @video_ns.doc(
+        params={
+            "authorization": {"in": "header", "description": "An authorization token"}
```

Comment:
Could you remove authorisation for now? We will provide api token at some point.
```suggestion
```